packages/utils/src/crypto/packet/constants.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Length of secp256k1 private keys resp. public keys
```

Comment:
what does resp. mean?

Reply:
changed the comment

Reply:
what does resp. mean?

Reply:
respectively

Reply:
let's stop using `resp.` because as a non native I find it confusing

Reply:
I mean, as a native speaker I find it confusing - it literally doesn't make sense in this context.